### PR TITLE
feat(esl-toggleable): add auto id

### PR DESCRIPTION
**Pull request by NastaLeo**
Add auto id to ESLToggleable component (will set if there isn't user's id).
User can also disable auto id.

### CODE REVIEW

File: src/modules/esl-toggleable/README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ Use `ESLToggleableDispatcher.init()` to initialize (and bind) ESLToggleableDispa
  - `body-class` - CSS class to add on the body element
  - `active-class` - CSS class to add when the Toggleable is active
  - `group` (`groupName`) - Toggleable group meta information to organize groups
+ - `no-auto-id` - Not to allow auto id
```
Comment (ala-n) [inline src/modules/esl-toggleable/README.md:24]:
```suggestion
 - `no-auto-id` - Disallow automatic id creation when it's empty
```

File: src/modules/esl-toggleable/core/esl-toggleable.shape.ts [hunk 2]
```diff
@@ -17,6 +17,8 @@ export interface ESLToggleableTagShape<T extends ESLToggleable = ESLToggleable>
   /** Define Toggleable group meta information to organize groups */
   'group'?: string;
 
+  /** Disable auto id */
```
Comment (ala-n) [inline src/modules/esl-toggleable/core/esl-toggleable.shape.ts:20]:
```suggestion
  /** Disallow automatic id creation when it's empty */
```

File: src/modules/esl-toggleable/core/esl-toggleable.ts [hunk 3]
```diff
@@ -57,6 +58,8 @@ export class ESLToggleable extends ESLBaseElement {
   /** Selector to mark inner close triggers */
   @attr({name: 'close-on'}) public closeTrigger: string;
 
+  /** Not to allow auto id */
```
Comment (ala-n) [inline src/modules/esl-toggleable/core/esl-toggleable.ts:61]:
```suggestion
  /** Disallow automatic id creation when it's empty */
```

File: src/modules/esl-toggleable/core/esl-toggleable.ts [hunk 4]
```diff
@@ -87,6 +90,9 @@ export class ESLToggleable extends ESLBaseElement {
 
   protected connectedCallback() {
     super.connectedCallback();
+    if (!this.id && !this.noAutoId) {
+      this.id = sequentialUID(ESLToggleable.is, 'esl-toggleable-');
```
Comment (ala-n) [inline src/modules/esl-toggleable/core/esl-toggleable.ts:94]:
```suggestion
      const tag = this.constructor.is;
      this.id = sequentialUID(tag, tag + '-');
```

**Review by ala-n — CHANGES_REQUESTED** [review]

**Comment by ala-n** [comment]
@NastaLeo it will be ideal if you update the previous commit with the more detailed comment (to simplify release process):
`feat(esl-togglable): add id auto-creation feature for toggleable instances`

**Review by abarmina — APPROVED** [review]

**Review by ala-n — APPROVED** [review]

**Review by julia-murashko — APPROVED** [review]

**Review by dshovchko — APPROVED** [review]

**Pull request merged**